packages/access-client/src/agent-use-cases.js

```diff
@@ -0,0 +1,157 @@
+import { Agent as AccessAgent, agentToData } from './agent.js'
```

Comment:
The inspiration for calling these 'use cases' is https://blog.cleancoder.com/uncle-bob/2012/08/13/the-clean-architecture.html#use-cases